### ISSUE TITLE
res_agi seems slow to start up

### ISSUE THREAD

**Issue by ntadmin** [POSSA/Caller-ID-Superfecta#253]
I've been trying to speed up the time between the line ringing and the call ringing on a SIP phone. I'm using a raspberry pi, and so I obviously don't expect brilliant speed. However, when I call SIP phone to SIP phone locally, the connection is more or less instantaneous, so it an't be that bad.

Obviously, adding in superfecta will slow things down, hence the ability to control how long it waits etc.

What is interesting to me is that res_agi seems slow to start up. On a recent call log:

[2015-06-29 10:12:52.233] VERBOSE[23982][C-00000048] res_agi.c:     -- Launched AGI Script /home/neil/git/Caller-ID-Superfecta/agi/superfecta.agi
[2015-06-29 10:12:53.481] VERBOSE[23982][C-00000048] res_agi.c:  /home/neil/git/Caller-ID-Superfecta/agi/superfecta.agi: Executing Scheme..

And

[2015-06-29 10:12:53.865] VERBOSE[23982][C-00000048] res_agi.c:     -- Launched AGI Script /var/lib/asterisk/agi-bin/dialparties.agi
[2015-06-29 10:12:54.303] VERBOSE[23982][C-00000048] res_agi.c:  dialparties.agi: Starting New Dialparties.agi

In short: it took 0.25s to start up for superfecta and later is took 0.43s to start up for dialparties. This seems odd to me.

Clearly this is not a superfecta issue per se, but it is impacting on the performance of superfecta. Where should it be raised and iscussed?

**Comment by tm1000** [comment]
This is not something we will be looking into on a raspberry pi for numerous reasons. You will have to deal with the performance hit or not use superfecta

**Comment by ntadmin** [comment]
No worries, I wouldn't expect it to be. I am really asking (as I'm unclear on the question) where I should go to discuss the performance hit of res_agi - which 'sub system' is it part of. If you don't know, no worries, I'll keep looking.
